central/role/datastore/singleton.go

```diff
@@ -67,23 +67,34 @@ func Singleton() DataStore {
 
 type roleAttributes struct {
 	idSuffix           string
+	postgresID         string
 	description        string
 	resourceWithAccess []permissions.ResourceWithAccess
 }
 
+func (attributes *roleAttributes) getID() string {
+	if env.PostgresDatastoreEnabled.BooleanSetting() {
+		return attributes.postgresID
+	}
+	return rolePkg.EnsureValidPermissionSetID(attributes.idSuffix)
+}
+
 var defaultRoles = map[string]roleAttributes{
```

Comment:
nit: might be worthwhile, like we do with the tags in our proto message definition, to create a comment with `// Next ID: xxxxx` so other people have it easier in adding a new default role.